build/pipelines/azure-pipelines.loc.yaml

```diff
@@ -51,4 +51,4 @@ jobs:
     condition: eq(variables['hasChanges'], '1')
     inputs:
       artifactName: Patch
-      targetPath: $(Build.ArtifactStagingDirectory)
\ No newline at end of file
+      targetPath: $(Build.ArtifactStagingDirectory)
```

Comment:
The LocalizedStrings.patch is stored at $(Build.ArtifactStagingDirectory). Would removing the targetPath here affect this task's goal to publish patch file as artifact?

Reply:
Oops, please ignore my comment above. 😂 It looks good to me .